README.md

```diff
@@ -465,7 +465,8 @@ install [CUDA](https://developer.nvidia.com/cuda-downloads) and
 
 JAX provides pre-built CUDA-compatible wheels for **Linux x86_64 only**. Other
 combinations of operating system and architecture are possible, but require
-[building from source](https://jax.readthedocs.io/en/latest/developer.html#building-from-source).
+[building from source](https://jax.readthedocs.io/en/latest/developer.html#building-from-source)
+or [JAX Toolbox bleeding edge containers](https://github.com/NVIDIA/JAX-Toolbox).
```

Comment:
Maybe we can create a separate section for this?

SHould this be tested before recommending to users?

Reply:
I amended the commit to have a new section.

I also specify what kind of containers they are.
We have a nightly CI in JAX-Toolbox, so they are tested. But they are clearly bleeding edge container and hopefully this is clear in the new version.